dashboard/app/models/reference_guide.rb

```diff
@@ -0,0 +1,70 @@
+require 'fileutils'
+
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  name              :string(255)
+#  slug              :string(255)
+#  course_version_id :integer
+#  content           :text(65535)
+#  order             :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+class ReferenceGuide < ApplicationRecord
+  belongs_to :course_version
+
+  validates_uniqueness_of :slug, scope: :course_version_id
+
+  def serialize
+    {
+      id: id,
+      name: name,
+      slug: slug,
+      course_version_id: course_version_id,
+      content: content,
+      order: order
+    }
+  end
+
+  def write_serialization
+    return unless Rails.application.config.levelbuilder_mode
+    file_path = Rails.root.join("config/reference_guides/#{slug}.json")
+    object_to_serialize = serialize
+    dirname = File.dirname(file_path)
+    unless File.directory?(dirname)
+      FileUtils.mkdir_p(dirname)
+    end
+    File.write(file_path, JSON.pretty_generate(object_to_serialize))
+  end
+
+  def self.seed_all
+    removed_records = all.pluck(:id)
+    Dir.glob(Rails.root.join("config/reference_guides/**/*.json")).each do |path|
```

Comment:
this does not match the path above

Reply:
doesn't it? is there something i'm missing about ruby file glob syntax?

Reply:
I think the /**/ means a directory but I could be wrong

Reply:
it works recusively, but also matches no directory! here's an example from the Dir.glob docs:
```
rbfiles = File.join("**", "*.rb")
Dir.glob(rbfiles)                   #=> ["main.rb",
                                    #    "lib/song.rb",
                                    #    "lib/song/karaoke.rb"]```

Reply:
Ahh ok 👍